### PR TITLE
(CT-09):delete post api integration

### PR DESCRIPTION
**Pull request by haojha**
Before Deleting on hovering the first card
MicrosoftTeams-image (28)

After clicking the delete icon on 1st card ,it is removed:
MicrosoftTeams-image (29)
Toast message screenshot :
MicrosoftTeams-image (31)

Things covered:
- **Added Delete Icon in the cards**
- **API integration for deleting the individual post**
- **Added tooltip on delete icon**
- **Added toast message after successful deletion**

### CODE REVIEW

**Review by Bhu5han — APPROVED** [review]
Looks great!

File: src/utils/apis/api.js [hunk 1]
```diff
@@ -15,4 +15,20 @@ const fetchCommentsByPostID = async (postId) => {
...
\ No newline at end of file
+  //delete post by id
+  export const deletePostByID = async (postId) => {
+    try {
+      const response = await fetch(`https://dummyjson.com/posts/${postId}`, {
+        method: 'DELETE',
+      });
+  
+      if (response.ok) {
+        console.log('Post deleted successfully');
```
Comment (supjain) [inline src/utils/apis/api.js:26]: Logs are not needed here. Remove it from everywhere.

Comment (Bhu5han) [reply src/utils/apis/api.js]: Changes done earlier.

**Review by koushik-singh — APPROVED** [review]
LG.

**Review by supjain — APPROVED** [review]

**Pull request merged**